src/hw-monitor.h

```diff
@@ -266,7 +119,12 @@ namespace librealsense
         }
     };
 
-    std::string hwmon_error_string( command const &, hwmon_response e );
+    typedef int32_t hwmon_response;
+    class base_hwmon_response_handler { // base class for different product number to implement responses
+    public:
+        inline virtual std::string hwmon_error2str(int e) const = 0;
+        virtual hwmon_response hwmon_Success() const = 0;
```

Comment:
Meaning is ambiguous, can be understood as `is_successful` or `success_value`.
According to the rest of the code it's the later, please rename.
Note - naming convention, should not capitalize after underscore.